packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { adminTenantId } from '@logto/schemas';
+import { adminTenantId, maxFreeTenantNumbers } from '@logto/schemas';
```

Comment:
nit: I prefer `maxFreeTenantLimit` or `maxFreeTenantCount`

Reply:
I prefer the first one `maxFreeTenantLimit` :P